File: app/src/main/java/com/antonio/samir/meteoritelandingsspots/util/analytics/AnalyticsUtil.java
package com.antonio.samir.meteoritelandingsspots.util.analytics;

import android.content.Context;


public class AnalyticsUtil {

    private static final Analytics analytics;

    static {
        analytics = new GoogleAnalyticsUtil();
    }

    public static void start(Context context) {
        analytics.start(context);
    }

    public static void logEvent(String category, String event) {
        analytics.logEvent(category, event);
    }

}

File: app/src/main/java/com/antonio/samir/meteoritelandingsspots/util/analytics/AnalyticsUtil.kt
package com.antonio.samir.meteoritelandingsspots.util.analytics

import android.content.Context


object AnalyticsUtil {

    private val analytics: Analytics

    init {
        analytics = GoogleAnalyticsUtil()
    }

    fun start(context: Context) {
        analytics.start(context)
    }

    fun logEvent(category: String, event: String) {
        analytics.logEvent(category, event)
    }

}

